src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -457,11 +500,46 @@ private void release()
             }
         }
 
+        private long challenge(
+            long now,
+            long traceId,
+            long delay,
+            LongConsumer doChallenge)
+        {
+            if (withinChallengeResponseTimeout(now))
```

Comment:
Inline `withinChallengeResponseTimeout` method body here and reduce to single `challengeAfter()` call.
If `challengeAfter()` is not used elsewhere, then also inline `challengeAfter()` method here.